week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -73,6 +101,58 @@ function insertShoppingList(shoppingList) {
  */
 function insertBooks(books) {
   //Write your code in here
+  let content = document.querySelector("#content");
+  let list = document.createElement('ul');
+
+  for(let i = 0; i < books.length; i++){
+    let infoEls = document.createElement('li');
```

Comment:
```suggestion
    let infoEl = document.createElement('li');
```
Consider using a singular here as you are creating a single `li` element that you then append to the `ul`